lego/apps/events/views.py

```diff
@@ -250,6 +255,30 @@ def allergies(self, request, *args, **kwargs):
         event_data = serializer(event).data
         return Response(event_data)
 
+    @decorators.action(detail=True, methods=["GET"])
+    def statistics(self, request, *args, **kwargs):
+        event_id = self.kwargs.get("pk", None)
+        event = Event.objects.get(pk=event_id)
+
+        headers = {
+            "Authorization": f"Bearer {settings.PLAUSIBLE_KEY}",
+            "Content-Type": "application/json",
+        }
+
+        created_at = event.created_at.strftime("%Y-%m-%d")
+        now = datetime.now().strftime("%Y-%m-%d")
+        # Plausible wants the date on this schema: YYYY-MM-DD,YYYY-MM-DD
+        date = f"{created_at},{now}"
```

Comment:
How will this affect events created a long time ago? (In terms of amount of data etc)

Reply:
Ye that's a problem, but I'm planning on giving the user the ability to filter on different timeseries. This is still a nice mvp though, since we don't have plausible analytics for old events.